nibabel/freesurfer/tests/test_io.py

```diff
@@ -54,26 +55,43 @@ def test_geometry():
     assert_equal(0, faces.min())
     assert_equal(coords.shape[0], faces.max() + 1)
 
-    # Test quad with sphere
     surf_path = pjoin(data_path, "surf", "%s.%s" % ("lh", "sphere"))
-    coords, faces = read_geometry(surf_path)
+    warnings.filterwarnings('always', category=DeprecationWarning)
```

Comment:
Doesn't this need a `with` block to prevent the warning being ignored outside these tests?